apps/site/src/components/Footer/Footer.tsx

```diff
@@ -19,22 +20,27 @@ const SOCIALS: Social[] = [
 	{
 		icon: Hack,
 		link: "https://hack.ics.uci.edu/",
-		alt: "Hack at UCI",
+		alt: "Hack at UCI logo that links to Hack at UCI's club website",
```

Comment:
Thought [repeating]: while I appreciate the considerations for accessibility, these seem a bit verbose: a screen reader would already announce that the image is a link.
> The text alternative for [functional images] should convey the action that will be initiated (the purpose of the image), rather than a description of the image ([W3C WAI's Tutorial on Functional Images](https://www.w3.org/WAI/tutorials/images/functional/))

Perhaps consider having just "Hack at UCI's Club Website", "Hack at UCI's Facebook Page" etc.